templates/layouts/custom-gift-amount.liquid

```diff
@@ -131,14 +117,24 @@
       _.pick(champaign.personalization.location, 'currency')
     );
 
+    var countryLanguage = `${champaign.page.language_code}-${champaign.personalization.location.country}`;
+    
```

Comment:
Not sure we can rely all the time on this. I'd suggest we hardcode a list of countries that we support country-specific localization for, and if it's not one of them, we fall back to just the `champaign.page.language_code`. This is mainly because I might be visiting a spanish page but my country will be GB, and there's no `es-GB` so we'd want to fall back to just `es`.

For now let's say we support:

```
en-US
en-GB
en-CA
en-NZ
en-AU
en-IE
de-DE
de-AT
de-CH
fr-FR
fr-CA
es-ES
es-AR
es-MX
```

everything else should just default to the page language.

Reply:
Okie @eyko  Ill make that change